src/extensions/menu.extension.ts

```diff
@@ -17,8 +20,32 @@ export class Menu {
     options?: {
       level?: number;
       headline?: string;
+      showHelp?: boolean;
+      helpLabel?: string;
+      backLabel?: string;
+      cancelLabel?: string;
+      byeLabel?: string;
```

Comment:
For `bye`, message fits better than `label`, since it is not the label of a menu item but an output message.

Reply:
Yes, this makes sense.
Updated!